docs/UserGuide.md

```diff
@@ -154,9 +156,10 @@ Expected outputs when command fails:
 Deletes the specified patient or an optional fields of the patient from HealthSync.
 
 * Deletes the patient or an optional field of the patient with the specified `n/NAME or id/IC_NUMBER`.
```

Comment:
Can I clarify how we are handling `or`s in our UG? I remember seeing this issue being handled by you during our meeting, will it be in a separate PR?

Reply:
It will be resolved in a separate PR.